### DISCUSSION TITLE
Github Feed: allow us to turn stuff off

### DISCUSSION THREAD

**Discussion by thrashcardiom** [community/community#30591] [Other Feature Feedback, Questions, & Ideas]
My 'feed' seems to be a list of people I'm recommended to follow.  I have absolutely no interest in this facility.  I should be able to turn off the "recommended people" list so I never see it.

The fact that I don't follow _anyone_ yet this feed is based on "people you follow" makes its recommendations worthless.

**Comment by depascalis** [comment]
Elaborating on this. I'd like the _Recommendations_ filter to be divided into more granular options. Personally I'm not interested in _Recommended based on people you follow_ , I'm however interested in repositories that are listed with the heading _Recommendations for you_. Currently, turning the _Recommendations_ filter off will hide both of the examples above.

_Edit_
Found another issue which exactly describes this, mockup included https://github.com/orgs/community/discussions/19287#discussion-4168318

**Comment by OlgaGrom** [comment]
Hi
